### ISSUE TITLE
New Field: Specific Teaching & Learning Resources

### ISSUE THREAD

**Issue by scottofletcher** [participedia/api#414]
See row 49 in the [Case Fields Editor XYZ spreadsheet](https://docs.google.com/spreadsheets/d/1zAkNVa-mjMsMqtIrjYL-j96fP4LTGOTFjs668WwxxRI/edit?ts=5c3d0575#gid=1762179507)

Database key: specific_learning_resources
XYZ label: Specific Teaching & Learning Resources 
Instructional text: What specific teaching & learning resources were used in this case? Type to select from entries already in the Participedia database. 
Info bubble: If you cannot find your teaching & learning resource, first publish this entry to save your work. Next add a new method, tool or technique entry. Then return to this case entry and add your new method, tool, or technique. 
Placeholder Type to search & generate possible database matches
Values: All teaching & learning resources on Participedia

**Comment by plscully** [comment]
@scottofletcher  @jesicarson @dethe  Now that I look at the instructions in writing I again have doubts that this is a good idea. Very few cases are going to use teaching & learning resources that are already on PPedia. Background briefing materials are usually specifically tailored to a particular case.  Assuming I'm right about this, users are almost always going to think they need to find the resource outside of our site and then add it to the TLRC. Perhaps the chance that this will discourage users from continuing can be mitigated by different instructions.  ... Even better, perhaps instead we simply include advice in the guidelines and in the info bubble that goes with media/files that recommends users upload to the case entry any informational or background materials used in the case. When appropriate our admins/editors can then re-purpose those files and add them to TLRC.

Also, I don't think you meant to include the text "Next add a new method, tool or technique entry. Then return to this case entry and add your new method, tool, or technique."

**Comment by dethe** [comment]
@plscully Are you saying that having this field is a bad idea or that the wording needs work? If it is the first, the case model is locked until we have all features completed, so we will have to live with it for now. Once we've finished implementing everything we can address model issues again as needed. If it is just wording, we'll soon have the static edit pages in place and we won't need issues to fix them.

**Comment by plscully** [comment]
@dethe It's a little of both, and it won't be a problem if the field is in there for while, even until after P1. That will give us a chance to see how users respond to it.

**Comment by plscully** [comment]
Sorry, I didn't mean to close this. That was an accident.

**Comment by scottofletcher** [comment]
@plscully Yes, I see what you mean about users thinking they have to enter content into the TLRC (in fact, that's kind of what we want them to thing -- and do -- with the Specific Methods and Specific Tools & Techniques fields) but, at the same time, we want to encourage people to enter cases on the user of TLRC content. I think what you're proposing is the best idea: create an info bubble for the 'Files' field that includes things like briefing materials, teaching & learning resources, facilitator guidelines. etc. The question is whether or not we want the TLRC materials to be linked to cases. Edana and Sule's memo on data collection suggested that the site "connect participatory processes to their respective “methods,” and to the organizations that organized the processes, so they can be appended into a single usable dataset." We've done that with cases, methods, tools & techniques, and organizations so my feeling is that this functionality should extend to TLRC entries. 
Three ways we might do this without overloading the average user:
1 - include it as an accordion field that only opens if the user selects a General Type of Method or General Type of Tool/Technique that would use a TLR. For example, "Experiential and immersive education (i.e. alternative approaches to teaching and learning)" (this is imperfect because 1) users might skip this field, and 2) TLRs could, conceivably, be used with any type of method/tool/technique)
2 - include it as an according field that only opens if the user selects "Teaching/Instructing" under "Types of Interaction Among Participants" 
3 - include as an according field that opens if the user answers 'Yes' to a simple Y/N question like "Did this case involve teaching & learning?" 

RE "Next add a new method, tool or technique entry. Then return to this case entry and add your new method, tool, or technique": yes, my mistake. I clearly didn't re-purpose those fields very well!

@dethe If the case model is locked that's fine, let's go with Pat's suggestion and not fiddle with things. I'll go through all the docs where I included this field and get rid of it (or strike it through and include the note 'TBD')
